### DISCUSSION TITLE
v5.0.0

### DISCUSSION THREAD

**Discussion by voipiran** [voipiran/VOIZ#6] [General]
- Add WebRTC Webphone
- Add Queue Survey
- Add 2 types of ChanSPY
- Add help Menu
- Add HTOP linux App
- Add SNGREP Sip Packet Monitoring App
- New Login Interface
- New Logo

<hr /><em>This discussion was created from the release <a href='https://github.com/voipiran/VOIZ/releases/tag/'>v5.0.0</a>.</em>

**Comment by mforooghii** [comment]
what is the default password of vtiger?
